app/admin/dashboard.rb

```diff
@@ -163,5 +163,40 @@
         end
       end
     end
+
+    columns do
+      columns do
+        column do
+          panel(link_to(I18n.t('admin.no_invoice_job_users.title'), admin_job_users_path)) do # rubocop:disable Metrics/LineLength
+            scope = JobUser.order(created_at: :desc).
+                    includes(:user,
+                             job: %i(translations language)).
+                    where(will_perform: true,
+                          application_withdrawn: false,
+                          'jobs.job_end_date': Time.at(0).utc..Time.now.utc,
+                          'frilans_finans_invoices.id': nil).
+                    left_joins(:frilans_finans_invoice).
+                    limit(20)
```

Comment:
Wow, I'm impressed! You really got the hang of the query interface fast 👑 

We could extract this query to a method/scope on `JobUser` (the limit call should be left here though).

Nevertheless 👍

Reply:
Ah I see 👍 Just to get a better understanding I have some questions.

If I refactor out the query how should i handle the includes?
Reusing the query probably does not imply that you would like to include all that stuff.
However as I understand it the jobs include is needed (but not with the translation stuff) since it is generated into the SQL FROM clause and is needed in the rest of the query. (unlike the user and the translated data.)
Hence I can not skip the include and just refactor out the where and left join. (it would not be standalone) (would always require a jobs include or a join before)